apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -311,17 +315,10 @@ private async Task<int> ExecuteMutationsAsync(CancellationToken cancellationToke
 
             private List<Mutation> GetMutations()
             {
-                // Currently, ToProtobufValue doesn't use the options it's provided. They're only
-                // required to prevent us from accidentally adding call sites that wouldn't be able to obtain
-                // valid options. For efficiency, we just pass in null for now. If we ever need real options
-                // from the connection string, uncomment the following line to initialize the options from the connection.
-                // SpannerConversionOptions options = SpannerConversionOptions.ForConnection(SpannerConnection);
-                SpannerConversionOptions conversionOptions = null;
-
                 // Whatever we do with the parameters, we'll need them in a ListValue.
                 var listValue = new ListValue
                 {
-                    Values = { Parameters.Select(x => x.SpannerDbType.ToProtobufValue(x.GetValidatedValue(), conversionOptions)) }
+                    Values = { Parameters.Select(x => x.GetConfiguredSpannerDbType(ConversionOptions).ToProtobufValue(x.GetValidatedValue())) }
```

Comment:
This is another place we *could* extract ConversionOptions to a local variable (just above the declaration of `listValue`) to avoid calling evaluating it more than once - but again, that may well be premature optimization anyway.